### PR TITLE
[DS-1163] Add a class for location's cards to make it in equal height

### PR DESCRIPTION
**Pull request by aleevas**
[DS-1163](https://yusa.atlassian.net/browse/DS-1163)

LocationСardEquialHeight

### CODE REVIEW

File: modules/openy_map_lb/openy_map_lb.install [hunk 1]
```diff
@@ -12,3 +12,24 @@ function openy_map_lb_update_9001(&$sandbox) {
...
+    'views.view.locations_lb' => [
+      'display',
+    ],
+  ];
+
+  $config_updater = \Drupal::service('openy_upgrade_tool.param_updater');
+  foreach ($configs as $config_name => $params) {
+    $config = $config_dir . $config_name . '.yml';
+    foreach ($params as $param) {
+      $config_updater->update($config, $config_name, $param);
```
Comment (podarok) [inline modules/openy_map_lb/openy_map_lb.install:32]: can we update only the section of the view config, not all view?
I'm caring about overriding someone's customisation

**Review by podarok — CHANGES_REQUESTED** [review]
let's usee param.updater not full config update @aleevas

File: modules/openy_map_lb/openy_map_lb.install (hunk 1 above)
Comment (aleevas) [reply modules/openy_map_lb/openy_map_lb.install:32]: @podarok fixed

Comment (podarok) [reply modules/openy_map_lb/openy_map_lb.install:32]: tnx

**Pull request merged**